content/help/manual/create_account/index.md

```diff
@@ -3,106 +3,77 @@ title: Creating An Account
 weight: 2
 icon: user-plus
 anchors:
-    - hash: step-1-enter-your-name-and-email-address
-      title: Enter Your Name And Email
-    - hash: step-2-confirm-your-email-address
-      title: Confirm Your Email Address
-    - hash: step-3-choose-your-master-password
+    - hash: step-1-download-the-app-or-visit-login-page
+      title: Download the App or visit the Login Page
+      hash: step-2-enter-your-email-address
+      title: Enter Your Email Address
+    - hash: step-3-email-authentication
+      title: Email Authentication
+    - hash: step-4-choose-your-master-password
       title: Choose Your Master Password
 ---
+{{< figure caption="Creating Your account?" >}}
+{{< iframe src="1_sign_up_process.mp4" width="640" height="350" >}}
+{{< /figure >}}
 
-Creating a Padloc account is simple and takes only a few moments. To get
-started, you can either open the Padloc web app by going to
-https://web.padloc.app or by downloading and installing the app. The first
-thing you'll see is a login screen. Since you don't have an account yet, you
-can't login at this point. Instead, click on **Sign Up Now**.
+## **Step 1: Download the [App](https://padloc.app/downloads/) or visit the [Login Page](https://web.padloc.app/signup)**
 
-## Step 1: Enter your name and email address
+## **Step 2: Enter Your Email Address**
```

Comment:
Please don't use `**` in headers. Headers should have a consistent look and if we want them to look more bold we can do that with CSS :)

Reply:
Have been removed.